File: barcosEnContienda/src/gestorAplicacion/Personas/CapitanPirata.java
package gestorAplicacion.Personas;

import gestorAplicacion.Embarcaciones.Barco;

public class CapitanPirata extends Capitanes {
	
	public CapitanPirata(String apodo, Barco barco, int liderazgo, int elocuencia, Tripulacion tripulacion) {
		super(apodo, barco, liderazgo, elocuencia, tripulacion);
	}
	
	public void yells() {
		System.out.print("¡Ahoy!");
	}
}

File: practica-2-grupo-1-equipo-7-Barcos/src/gestorAplicacion/personas/capitanPirata.py
from gestorAplicacion.embarcaciones.barco import Barco
from gestorAplicacion.personas.capitanes import Capitanes

class CapitanPirata(Capitanes):
    
    def __init__(self,apodo,liderazgo,elocuencia):
        super().__init__(apodo,liderazgo,elocuencia)

    def yells(self):
        return "Ahoy! \n"